diff --git a/node_modules/@klinecharts/pro/dist/klinecharts-pro.js b/node_modules/@klinecharts/pro/dist/klinecharts-pro.js
index 6f78d0e..8843c3e 100644
--- a/node_modules/@klinecharts/pro/dist/klinecharts-pro.js
+++ b/node_modules/@klinecharts/pro/dist/klinecharts-pro.js
@@ -3389,7 +3389,10 @@ const Dl = (e) => {
     setSymbol: p,
     getSymbol: () => y(),
     setPeriod: M,
-    getPeriod: () => L()
+    getPeriod: () => L(),
+    setCustomApi: (h) => {
+      n == null || n.setCustomApi(h);
+    }
   });
   const c1 = () => {
     n == null || n.resize();
@@ -3443,6 +3446,10 @@ const Dl = (e) => {
               return E === R1.XAxis ? B.formatDate(f, v, "MM-DD HH:mm") : B.formatDate(f, v, "YYYY-MM-DD HH:mm");
             case "day":
             case "week":
+              if (E === R1.XAxis) {
+                const h1 = new Date(v), f1 = h1.getDate();
+                return f1 === 1 ? h1.toLocaleString("en-US", { month: "short" }) : String(f1);
+              }
               return B.formatDate(f, v, "YYYY-MM-DD");
             case "month":
               return E === R1.XAxis ? B.formatDate(f, v, "YYYY-MM") : B.formatDate(f, v, "YYYY-MM-DD");
